pkg/github/github.go

```diff
@@ -9,11 +9,37 @@ import (
 	"net/url"
 	"strings"
 
+	"github.com/bradleyfalzon/ghinstallation/v2"
+
 	"github.com/google/go-github/v53/github"
 	"golang.org/x/crypto/nacl/box"
 	"golang.org/x/oauth2"
 )
 
+type Config struct {
+	GithubURL            string
+	GithubToken          string
+	GithubPrivateKeyFile string
+	GithubAppID          int64
```

Comment:
nit: It being GitHub related is kind of implied as this is the `github` package. Suggest removing the `Github` prefix

```suggestion
	URL            string
	Token          string
	PrivateKeyFile string
	AppID          int64
```

Reply:
and done [a416038](https://github.com/jadolg/klum/pull/25/commits/a416038144738e20972b7554339a6c6450fa5b65)